production/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.20-alpine as builder
```

Comment:
I know this is a prior mistake but the `as` here generate a warning in docker `WARN: FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 1) `

Reply:
I fixed two instances of this in a new commit to this same PR.